test/db/archos/linux-x64/dbg_step_back

```diff
@@ -37,16 +37,16 @@ dk 9
 EOF
 EXPECT=<<EOF
 0x00400574
-0x00000000
 0x00400590
-0x00000003
 0x00400590
+0xfffffffffffff30e
```

Comment:
these values are simply not stable, as they may contain whatever data... so I'd say it is alright to just adjust the tests. In the long term we probably want to have better way to test debug tests.